scripts/create_gitlab_snapshot.py

```diff
@@ -67,6 +83,17 @@ def shutdown_instance(instance: Mapping[str, JSON]):
     log.info('Instance %s has stopped', instance_ids)
 
 
+def restart_instance(instance: Mapping[str, JSON]):
+    instance_ids = [instance['InstanceId']]
+    log.info('Restarting %s  GitLab instance', config.deployment_stage)
+    aws.ec2.start_instances(InstanceIds=instance_ids)
+    log.info('Waiting for the GitLab instance to start')
+    waiter = aws.ec2.get_waiter('instance_running')
+    waiter.wait(InstanceIds=instance_ids,
+                WaiterConfig=dict(MaxAttempts=9999, Delay=15))
+    log.info('Instance %s is running', instance_ids)
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#18string-interpolation

and this would log a list literal. That's an implementation detail. I would change `instance_ids` to `instance_id` and duplicate the list construction where needed.